db/models.py

```diff
@@ -1 +1,54 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
```

Comment:
In my opinion, it's better to write everything in one line where it's possible. If a line is too long then of course you need to split it into a few lines